packages/backend/src/util/validateSecretKey.ts

```diff
@@ -0,0 +1,9 @@
+export const validateSecretKey = (key: string) => {
```

Comment:
Let's make this an [assertion function ](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions)

```suggestion
export function assertValidSecretKey(val: unknown): asserts val is string {
    if(!val || typeof val !== "string") {
        throw ...
    }
}
```

Reply:
Done!